docs/docusaurus/src/components/WasThisHelpful/index.js

```diff
@@ -51,20 +52,24 @@ export default function WasThisHelpful(){
             $survey_id: '018dd725-c595-0000-00c6-6eec1b197fd0'
         })
         setIsOpen(false)
+        setError(false)
     }
 
-    const sendReview = (e) => {
+    const sendReview = async (e) => {
         e.preventDefault()
-        if(formData.description){
-            posthog.capture("survey sent", {
-                $survey_id: '018dd725-c595-0000-00c6-6eec1b197fd0',
-                $survey_response: formData.name,
-                $survey_response_1: formData.email,
-                $survey_response_2: formData.description,
-                $survey_response_3: pathname,
-                $survey_response_4: formData.selectedValue.replaceAll('-', ' ')
-            })
-            setIsOpen(false)
+        if (formData.description) {
+            const response = await fetch("/.netlify/functions/createJiraTicketInDocsBoard", {
```

Comment:
Same suggestion here, can we move this URL to a config file? we should also add a try-catch

Reply:
What do you mean by a config file, would you like me to add it somewhere else than a const? (i've changed that but i can take it to another file)

Reply:
Yup, you got it, I think moving it to a variable should work for now.